draft-fossati-tls-attestation.md

```diff
@@ -1061,6 +1075,22 @@ the TLS server will terminate the exchange.
 
 TBD.
 
+# Privacy Considerations {#priv-cons}
```

Comment:
To ground this section in more traditional privacy considerations, maybe add an initial sentence:

> Within this section, we assume that the Attester is a TLS client, which in turn represents an individual person. We are concerned about leakage of unnecessary information about that person such as correlation of different connections initiated by him or her.